public/usage-examples/graphics/draw_line_on_window/draw_line_on_window-1-simple-oop.cs

```diff
@@ -0,0 +1,30 @@
+using SplashKitSDK;
+
+namespace draw_line_on_window
+{
+    public class Program
+    {
+        public static void Main()
+        {
+            // Create Window
+            Window start = new Window("draw line on window", 600, 600);
+            SplashKit.ClearScreen(SplashKit.ColorBlack());

```

Comment:
`SplashKit.ColorBlack()` can be made more OO by using `Color.Black` instead.

And `SplashKit.ClearScreen` can be made more OO by using `window.Clear(...)`.